### DISCUSSION TITLE
Rotating columns header

### DISCUSSION THREAD

**Discussion by olisis** [Comcast/react-data-grid#2581] [Q&A]
Is it possible to rotate columns header by for e.g. by 90 degree?  I have a big number of columns with long name and just a status inside of the cell - so I would like to rotate columns header to save space.

Thanks, your help is really appreciated.

**Comment by barbara-lea** [comment]
Hi. It is really late to respond to this, but I am trying to do something similar.
I passed a higher `headerRowHeight` property to the `Grid` and in the column objects I also declared a `headerRenderer` property:
```tsx
headerRenderer: (props) => {
    const {
        column: { name },
    } = props;
    return (
        <span
            style={{
                transform: "rotate(-70deg)",
                display: "inline-block",
                width: "100%",
            }}
        >
            {name}
        </span>
    );
},
```

I also applied some additional styles to the header cells:
```css
.rdg-header-row .rdg-cell {
	background-color: white;
	border: none
}
```
Are there maybe some better approaches?
